security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,25 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-5-0"]
+== OpenShift Compliance Operator 1.5.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.5.0:
+
+* link:https://access.redhat.com/errata/RHBA-2024:3533[RHBA-2024:3533 - OpenShift Compliance Operator 1.5.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-5-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* As of this release, the Compliance Operator provides a unique profile ID for easier programmatic use.  (link:https://issues.redhat.com/browse/CMP-2450[*CMP-2450*])
+
+* With this release, the Compliance Operator provides PCI-DSS-NODE scans and CIS-NODE profile scans for ROSA HCP. (link:https://issues.redhat.com/browse/CMP-1544[*CMP-1554*])
```

Comment:
We could remove the `PCI-DSS-NODE` and `CIS-NODE` parts here since we're technically advertising all node profiles on ROSA HCP (not just PCI and CIS). Maybe something like:

"With this release, the Compliance Operator is supported and tested on ROSA HCP environments."

Reply:
Done, thank you!